docs/modules/ROOT/pages/documentation/microprofile/config/directory.adoc

```diff
@@ -1,3 +1,5 @@
+:source-highlighter: rouge
+
```

Comment:
```suggestion
```
Our Antora setup doesn't support this highlighter.

Reply:
Done within 94b4c36